packages/react-components/theme-designer/src/useThemeDesignerReducer.ts

```diff
@@ -53,72 +101,89 @@ export const useThemeDesignerReducer = () => {
     };
   };
 
-  const initialState: ReducerState = {
-    themeLabel: 'Teams Light',
-    brand: brandTeams,
-    theme: teamsLightTheme,
-    isDark: false,
-    overrides: {},
-  };
+  const [overrideState, dispatchOverrideState] = React.useReducer(overrideReducer, {
+    teamsLight: {},
```

Comment:
move the default values up into their own variable. put them above the reducer so the reader knows about them before looking at the reducer